app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-header.scss

```diff
@@ -19,10 +19,12 @@
   @include govuk-font(19, $weight: bold);
 }
 
+// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
   .gem-c-step-nav-header {
     padding: 0 0 govuk-spacing(4) 0;
-    background: none;
+    background: unset;
```

Comment:
What's the value of changing this to `unset` over `none`?

Reply:
I've been using `unset` as I think it communicates the rule change better. We're 'unsetting' any previously set backgrounds. I've used it again here for consistency with the other work on print styles. Using `none` would work just as well and might be more obvious though, especially as `unset` is fairly uncommon and not well known. Do you have a view on that?

Reply:
From my reading of the docs it's possible that `unset` might not be suitable in all situations, in that if the element already inherited from its parent, it would reset it back to that. We shouldn't be writing CSS like that (we use modifier classes rather than nesting inheritance) but it feels like `none` would be more bulletproof, particularly for edge cases.